Marlin/pins_PRINTRBOARD_REVF.h

```diff
@@ -24,14 +24,17 @@
  *  Rev B  2 JUN 2017
  *
  *  Converted to Arduino pin numbering
+ *  at90usb1286 to arduino pin mapping is here:
+ *     https://labitat.dk/wiki/Panelolu_and_Printrboard_the_easy_way
```

Comment:
The problem is there is some misinformation at that link, which is outdated and has shown to cause mounds of issues in the past.  If history hadn't proven this over and over again (that link you found just one example of dozens online, and in this tracker), it wouldn't be such a stickling point.

If I have some time I may take the ASCII version of a similar chart that PrintrbotCo has at the top of their fork's pins.h file and incorporate it (or you can here !), but for now it is surely best to omit links that cite the old pins numbering in any form.